packages/server/src/api/controllers/table/utils.ts

```diff
@@ -129,17 +129,19 @@ export function importToRows(
     // the real schema of the table passed in, not the clone used for
     // incrementing auto IDs
     for (const [fieldName, schema] of Object.entries(originalTable.schema)) {
+      const rowVal = Array.isArray(row[fieldName])
+        ? row[fieldName]
+        : [row[fieldName]]
       if (
         (schema.type === FieldTypes.OPTIONS ||
           schema.type === FieldTypes.ARRAY) &&
-        row[fieldName] &&
-        (!schema.constraints!.inclusion ||
-          schema.constraints!.inclusion.indexOf(row[fieldName]) === -1)
```

Comment:
Just to confirm - what happens here if there are no `constraints` or `inclusion` properties - I suppose it just creates an array with the `rowVal` in it, then gets the uniques from that?